inscriptions/ISic020036.xml

```diff
@@ -93,7 +94,7 @@
                                 <!-- Hourglass-shaped sign. Id: symbol02.-->
                                 <locus from="line1" to="line1">Line 1</locus>
                                 <dimensions type="letterHeight">
-                                    <height unit="mm"/>
+                                    <height unit="mm">11x25</height>
```

Comment:
is this dimensions of the symbol? it would be possible to add width as a separate element if you want (we don't display it at present, but there is no reason in priniciple not to record both (although it is awkward when a single symbol for comparability, since we won't be recording e.g. average module width in general).